valohai_yaml/utils/upload_store.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+from valohai_yaml.excs import ValidationError
+
+
+def parse_upload_store(upload_store: Optional[str]) -> Optional[str]:
+    if not upload_store:
+        return None
+    import re
+
+    if (isinstance(upload_store, str) and len(upload_store) <= 64) or re.match(
+        r"^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$",
```

Comment:
This is currently matched case-sensitively; an UUID would be just as valid written in upper-case.